nestkernel/growth_curve.h

```diff
@@ -77,6 +77,38 @@ class GrowthCurve
   const Name name_;
 };
 
+/* BeginDocumentation
+  Name: growth_curve_linear - Linear version of a growth curve
+
+  Description:
+   This class represents a linear growth rule for the number of synaptic elements
+   inside a neuron. The creation and deletion of synaptic elements when structural
+   plasticity is enabled, allows the dynamic rewiring of the network during the
+   simulation.
+   This type of growth curve uses an exact integration method to update the
+   number of synaptic elements: dz/dt = nu (1 - (1/epsilon) * Ca(t)),
+   where nu is the growth rate and epsilon is the desired average calcium
+   concentration.
+
+  Parameters:
+   eps          double -  The target calcium concentration (firing rate) that
+                          the neuron should look to achieve by creating or deleting
+                          synaptic elements.
```

Comment:
Dear @heplesser sorry for the confusion. I have rephrased that sentence so it is clear that the calcium concentration is an indirect measure of the firing rate. The value of nu is actually set in the SynapticElement class but I have extended its documentation in the description section.